.github/workflows/ci.yml

```diff
@@ -27,6 +35,13 @@ jobs:
         rust:
           - 1.36.0
           - nightly
+        os:
+         - ubuntu
+         - windows
+        exclude:
```

Comment:
May I ask the meaning of this exclusion?

Reply:
I removed that exclusion.
Originally, I excluded it because I had a lot of errors when I first ran it (I got the same error from both nightly and 1.36 respectively and it was a hassle), but, in reality, we probably don't need to exclude it.